lib/course_planner/offered_courses.ex

```diff
@@ -66,4 +66,12 @@ defmodule CoursePlanner.OfferedCourses do
     end)
     |> Enum.uniq_by(fn %{id: id} -> id end)
   end
+
+  def get_offered_courses_with_not_filled_class_attendances(date \\ Timex.now()) do
```

Comment:
what do you think of renaming this function?
maybe `with_pending_attendances`, remember that this is called with the module name (`OfferedCourses.with_pending_attendances()`), so we can avoid repeating it the module name in the function name